test/test_common/logging.h

```diff
@@ -61,6 +61,8 @@ class LogRecordingSink : public Logger::SinkDelegate {
   std::vector<std::string> messages_;
 };
 
+} // namespace Envoy
```

Comment:
Can you elaborate a bit on why we need to change this file? You mentioned the namespace change in quic_platform_test.cc requires this, but what exactly is the error?
I'd prefer to not to change this file if possible.

Reply:
LogLevelSetter, LogRecordingSink, ect. do not exist under namespace quic. If these macros are expanded under namespace quic, they need to be prefixed with `Envoy::`

Reply:
SGTM.
Can we only move the macros out of ::Envoy, and move StringPair and ExpectedLogMessages(below) back into ::Envoy?
A quick comment for EXPECT_LOG_CONTAINS_ALL_OF and EXPECT_LOG_NOT_CONTAINS, explaining why their body needs to used fully qualified envoy types, would be nice.

Reply:
Actually there is no need to move the macro's out of ::Envoy as long as those Envoy classes are used with Envoy::.